### ISSUE TITLE
fastify cors not working at nestjs@6.0.x

### ISSUE THREAD

**Issue by blueway** [nestjs/nest#1867]
file:  /platform-fastify/adapters/fastify-adapter.ts
```js
 public enableCors(options: CorsOptions) {
    this.register(cors, { options });
  }
```
should be:
```js
 public enableCors(options: CorsOptions) {
    this.register(cors, options );
  }
```

**Comment by kamilmysliwiec** [comment]
Would you like to create a PR?

**Comment by blueway** [comment]
#1881

**Comment by kamilmysliwiec** [comment]
Fixed in 6.0.5 :) Thanks for your help!
